docs/konnectors.md

```diff
@@ -320,18 +320,40 @@ On the host create the file /etc/rkt/net.d/10-containers.conf
 
 and run your container with the "--net=bridge" option. That way, a new interface is available in the container and gives you access to the host.
 
+## Konnector install and run details
+
+### Install
+
```

Comment:
I supposed it would be installed when the user says he wants to use it and each user would have its own installation of konnectors and I suppose it forces use to install it in VFS to make it work with several servers. 

But this raises the problem of running a konnector in VFS. Couldn't we copy the repository in a kind of sandbox directory which is not in VFS for each konnector run ?